Resources/config/imagine.xml

```diff
@@ -139,14 +141,18 @@
             <argument type="service" id="logger" on-invalid="ignore" />
         </service>
 
-        <!-- Controller -->
+        <service id="Liip\ImagineBundle\Service\FilterService" alias="liip_imagine.service.filter" />
```

Comment:
i'd only define such alias for autowiring services that we expect users of the bundle to need in their own services. not sure which of the services are expected to be used.
as said below, bundles should not use autowiring for their own services.

Reply:
yes, I'll leave it as it was